Book-A-Desk.Domain/Reservation.Command.BookADesk.fs

```diff
@@ -9,9 +9,9 @@ type BookADeskReservationCommand =
     }
 
 module BookADeskReservationCommand =
-    let provide getAllOffices =
+    let provide offices =
         let validate (command:BookADesk) reservationAggregate  =
-             BookADeskReservationValidator.validateCommand getAllOffices command reservationAggregate
+             BookADeskReservationValidator.validateCommand offices command reservationAggregate
```

Comment:
Instead of fetching the offices, we now inject them. Allowing the transition to offices in AmazonDB instead of in memory